packages/web3-eth-accounts/src/account.ts

```diff
@@ -77,3 +224,78 @@ export const create = (): {
 		encrypt,
 	};
 };
+
+/**
+ *  Decrypts a v3 keystore JSON, and creates the account.
+ *
+ * */
+export const decrypt = async (
+	keystore: Keystore | string,
+	password: string,
```

Comment:
We should be uniform in our support for buffers, we should handle a buffer being passed here as we do for `encrypt`:

```typescript
export const encrypt = async (
	privateKey: HexString,
	password: string | Buffer,
// ...
```